api/python/quilt3/packages.py

```diff
@@ -851,6 +875,8 @@ def manifest(self):
         for logical_key, entry in self.walk():
             yield {'logical_key': logical_key, **entry.as_dict()}
 
+
+    @ApiTelemetry("package.set")
```

Comment:
Per above, let's temporarily not instrument this call, as it could be high-volume.